spec/support/page_objects/trainees/contact_details.rb

```diff
@@ -0,0 +1,16 @@
+module PageObjects
+  module Trainees
+    class ContactDetails < PageObjects::Base
+      set_url "/trainees/{id}/contact-details/edit"
+      element :locale, ".govuk-radios.locale"
+      element :address_line_one, "input[name='trainee[address_line_one]']"
+      element :address_line_two, "input[name='trainee[address_line_two]']"
+      element :town_city, "input[name='trainee[town_city]']"
+      element :postcode, "input[name='trainee[postcode]']"
+      element :phone_number, "input[name='trainee[phone_number]']"
+      element :email, "input[name='trainee[email]']"
+      element :international_address, "input[name='trainee[international_address]']"
```

Comment:
Since user can only really have one or the other, feels like there is a lack of tests for `international_address` as well as for the `locale_code`.

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/74/commits/4d7b5ed57dd6b0032cc3f8c8afd3f061d4ea7cd4